app/src/main/res/values/strings.xml

```diff
@@ -78,18 +78,18 @@
 
     <string name="setting_mic_input">Mic input</string>
     <string name="setting_hotword_detection">Hotword Detection (Beta)</string>
-    <string name="setting_mic_key" translatable="false">micInput</string>
+    <string name="setting_mic_key" translatable="false">mic_input</string>
```

Comment:
@iamareebjamal I saw translations of these strings in other localization files, which I think should not be there because the Lint tool does not need to look for its translation in any of the res files.
If 'false' has been assigned to the translatable attribute of a string resource, is there a need to add the translations to these strings in other localization files?

Reply:
No

Reply:
👍 Will be removing unnecessary translations.